evaluation.md

```diff
@@ -5,15 +5,39 @@
+
 ## External DSL
 
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+I am proud to say that my external DSL was able to correctly run my original example programs without modification. Well, with one small modification: the external DSL actually revealed two bugs in my original code (I refered to the state moveUp with moveUP, so the names didn't match). After I fixed that, it ran just fine. This is exciting because it means that I was able to get the more "advanced" functionality of the DSL working. So, no quotes around variable names, inference of the start state and state transitions, and chained logic (stuff like "cannot move up or left or right" is equivalent to "cannot move up and cannot move left and cannot move right"). I think having these features (or lack of syntax) is a nice "feature" of my DSL, so it was great to see it working. I'm also happy that I was able to easily get the full natural language syntax working. I think it really helps with readability and ease of use.
+
+There was one or two things that I didn't implement that may have been expected of the original language. First of all, the lanugage does not support weird cases where you specify no conditions (so, you don't care whether any of the tiles around you are blocked). While this may be desirable if you wanted to exploit the order in which the rules were evaluated, this could also make code harder to follow (since you now have to figure out what you can infer from previous conditions). If you're moving, you should check to make sure you can actually move in that direction. If you're not moving and you're not checking anything around you, then why not elimiate the current state entirely or transition directly to the next state? If you _really_ want to do this, you can always write two rules that accomplish the same task (which is annoying, but still possible). My DSL also doesn't support infering that you do not want to move and you want to transition to the state you're already in. This would create an infinite loop, but it's still possible with my DSL (you just have to explicitly speficy that you're transitioning to the state you're in). Since these are generally behaviours you want to avoid, it's actually kind of a good thing that my DSL doesn't make them easy so that perhaps the programmer will realize that what they are doing is probably a bad idea. And, if they know what they're doing, they can still accomplish what they want.
+
```

Comment:
I love that you address the "What is hard to express in this language?" question and give good reasons for why they should be difficult! I agree!